tech-specs/src/specs/encoding.md

```diff
@@ -0,0 +1,15 @@
+# Encoding
+
+All the data fields are REQUIRED, unless specified otherwise.
+
+## The ledger
+
+### Transactions
+
+Transactions MUST be encoded using [proto3](https://developers.google.com/protocol-buffers/docs/reference/proto3-spec) in the format as defined for [message Tx](https://github.com/anomanetwork/anoma/blob/master/proto/types.proto).
```

Comment:
Can we inline the `.proto` syntax, perhaps?

Reply:
good idea! I've included the `.proto` file in the spec from source